### PR TITLE
[Docs] Update change log for V1.6.3

### PR DESCRIPTION
**Pull request by jillnogold**
https://iguazio.atlassian.net/browse/ML-6414

### CODE REVIEW

File: docs/change-log/index.md [hunk 1]
```diff
@@ -921,6 +952,7 @@ with a drill-down to view the steps and their details. [Tech Preview]
 
 | Will be removed|Deprecated|API                                                                                |Use instead                                                                                                                                                 |
 |---------------|------------|----------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| v1.8.0       |v1.6.3    |`FunctionSpec.clone_target_dir`                                                      |`ImageBuilder.source_target_dir`
```
Comment (liranbg) [inline docs/change-log/index.md]:
```suggestion
| v1.9.0       |v1.6.3    |`FunctionSpec.clone_target_dir`                                                      |`ImageBuilder.source_target_dir`
```

File: docs/install/remote.md [hunk 2]
```diff
@@ -23,8 +23,6 @@ Applications:
 - Python 3.9
 - Recommended pip 22.x+
 
-The MLRun server is based on a Python 3.9 environment. It's recommended to move the client to a Python 3.9 environment as well.
```
Comment (liranbg) [inline docs/install/remote.md]: why removed?

File: docs/change-log/index.md [hunk 3]
```diff
@@ -942,6 +967,7 @@ with a drill-down to view the steps and their details. [Tech Preview]
 
 | Will be removed|Deprecated|API                                                                                |Use instead                                                                                                                                                 |
 |---------------|------------|----------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| v1.9.0       |v1.6.3    |`FunctionSpec.clone_target_dir`                                                      |`ImageBuilder.source_target_dir`
```
Comment (liranbg) [inline docs/change-log/index.md]: uncertain it is needed. correct @alonmr ?

File: docs/change-log/index.md [hunk 4]
```diff
@@ -902,10 +931,6 @@ with a drill-down to view the steps and their details. [Tech Preview]
...
 |ML-5573|The default value of feature-set ingest() infer_options is "all" (which includes Preview) and as a result, during ingest, preview is done as well. As a result, if a validator was configured for a feature, each violation causes two messages to be printed.|NA|v1.6.0|
 |ML-5732|When using an MLRun client previous to v1.6.0, the workflow step status might show completed when it is actually aborted.|Abort the job from the SDK instead of from the UI, or upgrade the client. |1.6.0|
-|ML-5876|The maximum length of project name + the longest function name for `project.enable_model_monitoring` is 63 chars. |Keep the name combination at a maximum of 63 chars. |v1.6.0|
-|ML-6048|UI: An admin user cannot change its role in the project. | NA |v1.6.2|
-|ML-6045|UI: If a user-filter has a large number of matches, it may not display all the matching users. |Narrow your search to be sure you get all the matches.|v1.6.2|
-|ML-6051|UI: After an admin user deletes itself from a project, the user stays in the Projects Members page even though it has no permissions and cannot view any aspects of the project.|  NA |v1.6.2|
```
Comment (liranbg) [inline docs/change-log/index.md]: why removed?

Comment (jillnogold) [reply docs/change-log/index.md]: all of them moved to fixed except 5876, now put back in open

File: docs/install/remote.md (hunk 2 above)
Comment (jillnogold) [reply docs/install/remote.md]: returned

File: docs/change-log/index.md (hunk 3 above)
Comment (alonmr) [reply docs/change-log/index.md]: Users should no be using this directly but I think it's good to mention as they may want to change it manually in function yamls. @jillnogold please change `source_target_dir` to `source_code_target_dir`

File: docs/change-log/index.md [hunk 5]
```diff
@@ -13,6 +13,38 @@
 - [Deprecations and removed code](#deprecations-and-removed-code)
  
 
+## v1.6.3 (26 May 2024)
+
+### Workflows
```
Comment (alonmr) [inline docs/change-log/index.md:18]: maybe add https://iguazio.atlassian.net/browse/ML-4149

**Review by alonmr — COMMENTED** [review]
Missing tickets
https://iguazio.atlassian.net/browse/ML-5739
https://iguazio.atlassian.net/browse/ML-5772
https://iguazio.atlassian.net/browse/ML-6000
https://iguazio.atlassian.net/browse/ML-6026
https://iguazio.atlassian.net/browse/ML-6188
https://iguazio.atlassian.net/browse/ML-6194
https://iguazio.atlassian.net/browse/ML-6229
https://iguazio.atlassian.net/browse/ML-6317
https://iguazio.atlassian.net/browse/ML-6489

**Review by liranbg — APPROVED** [review]

**Pull request merged**